components/Card.tsx

```diff
@@ -0,0 +1,5 @@
+import styles from "../styles/Home.module.css";
+
+export default function Card({ name }) {
+  return <p className={styles.card}>{name}</p>;
```

Comment:
We are using Styled Components, please don´t start off without.

Reply:
you are right, I just used the default of create-next-app and added my styled components 😅